share/spice/doi/doi.js

```diff
@@ -0,0 +1,78 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_doi = function(api_result){
+
+        if (!api_result || !api_result.DOI || !api_result.author || !api_result.title || api_result.error) {
+            return Spice.failed('doi');
+        }
+
+        Spice.add({
+            id: "doi",
+
+            name: "Reference", // Or Science?
+            data: api_result,
+            signal: 'high',
+            meta: {
+                sourceName: "dx.doi.org",
+                sourceUrl: api_result.URL
+            },
+            normalize: function(item) {
+                return {
+                    url: item.URL,
+                    title: item.title,
+                    subtitle: sprintf( '%s, %s, DOI:%s', format_authors(item.author), item.issued.raw, item.DOI ),
+                }
+            },
+            templates: {
+                group: 'info',
+                options: {
+                    moreAt: true,
+                    moreText: {
+                        href: "/js/spice/doi_bibtex/" + api_result.DOI,
+                        text: 'BibTeX',
+                    }
```

Comment:
@jbarrett no effect! It's a style thing that I get a lot when I contribute to our code because I leave trailing commas as well. Just thought others should share my misery. :))